src/Components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends Component {
+  render() {
```

Comment:
While you did a nice job working on the project requirements I noticed you disabled eslint it is against microverse policy to disable linters because linters help us write clean code so it would be nice if you enabled eslint.

Reply:
Constructor is added as Linters are not allowed to add a class bases component without adding a constructor.